WMFComponents/Sources/WMFComponents/Components/Profile/ProfileView.swift

```diff
@@ -0,0 +1,48 @@
+import SwiftUI
+
+struct ProfileView: View {
```

Comment:
@l-olson1214 Can you rename this to WMFProfileView? We've been trying to prepend any public name that lives in WMFComponents with "WMF". This is just as an indication from callers in the app-side code (like ExploreViewController) that this component lives in one of the packages/frameworks.

Reply:
Can do!